### PR TITLE
Add logic to construct network URI.

### PR DESCRIPTION
**Pull request by quoctruong**
This is to address https://github.com/GoogleCloudPlatform/google-cloud-powershell/issues/458

---
This change is [Reviewable](https://reviewable.io/reviews/googlecloudplatform/google-cloud-powershell/461)

### CODE REVIEW

File: Google.PowerShell/Compute/GceCmdlet.cs [hunk 1]
```diff
@@ -218,6 +218,42 @@ public static string GetRegionNameFromUri(string uri)
...
+            }
+
+            if (string.IsNullOrWhiteSpace(project))
+            {
+                if (!networkName.StartsWith($"global/networks/"))
+                {
+                    networkName = $"global/networks/{networkName}";
+                }
+            }
+            else
```
Comment (iantalarico) [inline Google.PowerShell/Compute/GceCmdlet.cs]: else if?

File: Google.PowerShell/Compute/GceCmdlet.cs [hunk 2]
```diff
@@ -218,6 +218,42 @@ public static string GetRegionNameFromUri(string uri)
...
+            }
+
+            if (string.IsNullOrWhiteSpace(networkName))
+            {
+                networkName = "default";
+            }
+
+            if (string.IsNullOrWhiteSpace(project))
+            {
+                if (!networkName.StartsWith($"global/networks/"))
```
Comment (iantalarico) [inline Google.PowerShell/Compute/GceCmdlet.cs]: no need for $ here

File: Google.PowerShell/Compute/GceCmdlet.cs [hunk 3]
```diff
@@ -218,6 +218,42 @@ public static string GetRegionNameFromUri(string uri)
...
+            {
+                return networkName;
+            }
+
+            if (string.IsNullOrWhiteSpace(networkName))
+            {
+                networkName = "default";
+            }
+
+            if (string.IsNullOrWhiteSpace(project))
```
Comment (iantalarico) [inline Google.PowerShell/Compute/GceCmdlet.cs]: if (string.IsNullOrWhiteSpace(project) && !networkName.StartsWith("global/networks/"))?

**Comment by quoctruong** [comment]
@iantalarico PTAL

**Comment by quoctruong** [comment]
Thanks!

**Pull request merged**